internal/app/subpoll/poller.go

```diff
@@ -41,6 +41,8 @@ func Load(config *config.Config, bot *subgrok.Bot) *Poller {
 		Subscriptions: &subscription.Subscriptions{},
 	}
 
+	poller.TooRecent = make(map[string]bool)
```

Comment:
is there a reason this isn't initialised in the poller struct initialisation above?

Reply:
Resolved in 30c17e5.